src/couldBeSharedComponents/LayoutPanels/LayoutPanels.tsx

```diff
@@ -143,70 +170,81 @@ export const LayoutPanels = ({
     <button
       className={layoutPanelStyles.panelCloseButton}
       onClick={handleLeftToggleButtonClick}
+      aria-label="Toggle left panel"
     >
       {isLeftPanelOpenToUse ? "<" : ">"}
     </button>
   );
   const rightPanelButtonToUse = rightPanelToggleButton
     ? cloneElement(rightPanelToggleButton, {
         onClick: handleRightToggleButtonClick,
+        "aria-label": "Toggle right panel",
       })
     : internalRightPanelCloseButton;
 
   const leftPanelButtonToUse = leftPanelToggleButton
     ? cloneElement(leftPanelToggleButton, {
         onClick: handleLeftToggleButtonClick,
+        "aria-label": "Toggle left panel",
       })
     : internalLeftPanelCloseButton;
 
-  return (
-    <div className={layoutPanelStyles.layoutPanelsWrapper}>
-      {leftPanelContent ? (
-        <div
-          className={`${layoutPanelStyles.panel} ${leftPanelClassName ?? ""}`}
-          style={leftPanelDynamicStyles}
-          ref={leftPanelRef}
-        >
-          {isLeftPanelToggelable ? (
-            <div className={layoutPanelStyles.leftPanelCloseTab}>
-              {leftPanelButtonToUse}
-            </div>
-          ) : null}
-          {subpanelContent ? (
-            <PanelContentsWithSubpanel
-              isSubpanelOpen={isLeftSubpanelOpen!} // we ensure this is defined above
-              setIsSubpanelOpen={setIsLeftSubpanelOpen!} // we ensure this is defined above
-              subpanelContent={subpanelContent}
-              mainPanelContent={leftPanelContent}
-              isLeftPanelOpen={isLeftPanelOpenToUse}
-              subpanelClassName={subpanelClassName}
-            ></PanelContentsWithSubpanel>
-          ) : (
-            leftPanelContent
-          )}
```

Comment:
I realized this ternary wasnt necessary and is overly complex. In all cases now, regardless of it we are using a sub panel or not, we use the sub panel internal component and have it default to having the sub panel hidden (it shoudlnt have defaulted to having a sub panel open probably in the first place since Id expect most use cases would not have a sub panel initially open). This removes the unnecessary conditional (which was nested to boot!)